MAUI/ImageEditor/undo-redo.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: Undo Redo in .NET MAUI Image Editor Control | Syncfusion
```

Comment:
Undo Redo editing in .NET MAUI Image Editor Control | Syncfusion

Reply:
Changed a title